tests/test_cmd_lossless.sh

```diff
@@ -42,11 +42,11 @@ AVIFDEC="${BINARY_DIR}/avifdec"
 ARE_IMAGES_EQUAL="${BINARY_DIR}/tests/are_images_equal"
 
 # Input file paths.
-INPUT_Y4M="${TESTDATA_DIR}/kodim03_yuv420_8bpc.y4m"
+INPUT_PNG="${TESTDATA_DIR}/paris_icc_exif_xmp.png"
 # Output file names.
-ENCODED_FILE="avif_test_cmd_encoded.avif"
-DECODED_FILE="avif_test_cmd_decoded.png"
-DECODED_FILE_LOSSLESS="avif_test_cmd_decoded_lossless.png"
+ENCODED_FILE="avif_test_cmd_lossless_encoded.avif"
+DECODED_FILE="avif_test_cmd_lossless_decoded.png"
+DECODED_FILE_LOSSLESS="avif_test_cmd_lossless_decoded_lossless.png"
```

Comment:
Yannis: These tmp files are all created in `${TMP_DIR}`, which is unique. So the tmp file names should not need to be unique, right?

Reply:
Right, I confused the behavior with the .cc ones. I guess `/tmp` might still be given as an argument for `${TMP_DIR}` on some build systems so it does not hurt.